magento/pwa-studio
# [PWA-1974] Fix search suggested category double store code

## Description

Removing createHref from useSuggestedCategory since the URL is used in a Link component which also adds the basename.

## Related Issue







Closes [PWA-1974](https://jira.corp.magento.com/browse/PWA-1974)

## Acceptance





### Verification Stakeholders

@revanth0212 @supernova-at 

### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature

- Enable store code in URL
- Write "tanks" in search bar
- Select one of the categories (ex "Tanks in Blouses")
- Page should render without error (no Not Found page)

#### Is Browser/Device testing needed?

No

#### Any ad-hoc/edge case scenarios that need to be considered?

Current behaviour is that it copies the existing query string values. Could cause issues with "page" values out of range.

## Breaking Changes (if any)

None

## Checklist
